orchestration/dagster_orchestration/hca_orchestration/contrib/bigquery.py

```diff
@@ -33,6 +33,28 @@ def build_query_job(
 
         return query_job
 
+    def build_query_job(
```

Comment:
This is a helper that we (somewhat surprisingly) had not needed until now: fire off a simple query to BQ and return the job, no external destination or table overwrite.

Reply:
I understand that this new function allows us to perform a BQ query without having to build/autodetect an external config in the job_config and also doesn't write the results anywhere but I was a little confused about how this would work without a schema - is it just using the autodetect feature to do that?

Reply:
Good question. In this context, there is a schema; we are just firing a vanilla query off against an existing, vanilla dataset. BigQuery has another mode that allows you to query files stored in GCS (as we do in `build_query_job_using_external_schema`). We are setting `autodetect=true` there to avoid having to package up and ship a schema for those GCS areas.